### PR TITLE
KCI-235

### PR DESCRIPTION
**Pull request by DennisLammers**
feat(Docker): Add default user id and name to docker. This prevents issues when creating files through artisan or logs that are unaccessible by either the developer or the app. (Kudo's to @ibovangeffen for this fix.)

### CODE REVIEW

**Comment by ibovangeffen** [comment]
Kudos to @jspekken 😉

**Comment by DennisLammers** [comment]
> Kudos to @jspekken 😉

Haha, ook kudo's to @jspekken dan, dat wist ik niet :)

**Review by jspekken — DISMISSED** [review]

File: .docker/php8.0-apache/Dockerfile [hunk 1]
```diff
@@ -33,16 +35,25 @@ ENV APACHE_DOCUMENT_ROOT /var/www/public
...
 RUN sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/sites-available/*.conf
 RUN sed -ri -e 's!/var/www/!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf
 
 # Some aliasses.
-RUN echo 'alias pa="php artisan"' >> ~/.bashrc
-RUN echo 'alias phpunit="./vendor/bin/phpunit"' >> ~/.bashrc
-RUN echo 'alias refresh="pa migrate:fresh"' >> ~/.bashrc
+RUN echo 'alias pa="php artisan"' >> /home/king/.bashrc
+RUN echo 'alias phpunit="./vendor/bin/phpunit"' >> /home/king/.bashrc
+RUN echo 'alias refresh="pa migrate:fresh"' >> /home/king/.bashrc
```
Comment (koenhoeijmakers) [inline .docker/php8.0-apache/Dockerfile]: I'd use `/home/$user/` here given you're already doing the same on lines [38-42] above

File: docker-compose.yml [hunk 2]
```diff
@@ -35,7 +35,11 @@ services:
       - default
 
   app:
-    build: .docker/php8.0-apache/
+    build:
+      context: .docker/php8.0-apache/
+      args:
+        user: king
+        uid: 1000
```
Comment (koenhoeijmakers) [inline docker-compose.yml:42]: Do know that this only works for systems that have 1000 as the first uid (like ubuntu)

File: .docker/php8.0-apache/Dockerfile (hunk 1 above)
Comment (DennisLammers) [reply .docker/php8.0-apache/Dockerfile]:
```suggestion
RUN echo 'alias pa="php artisan"' >> /home/$user/.bashrc
RUN echo 'alias phpunit="./vendor/bin/phpunit"' >> /home/$user/.bashrc
RUN echo 'alias refresh="pa migrate:fresh"' >> /home/$user/.bashrc
```

File: docker-compose.yml (hunk 2 above)
Comment (DennisLammers) [reply docker-compose.yml:42]: Good to know, but since it works in `php8.0-apache`, guess we'll see if it keeps working on other (newer) images ;)

**Review by pretendings — APPROVED** [review]

**Pull request merged**